CONTRIBUTING.md

```diff
@@ -196,6 +196,14 @@ Note: If you install Python on Windows using the MSI installer, it is highly rec
 
 Windows typically doesn't have the *make* tool installed. Windows users without *make* should look at the "detailed setup" section below.
 
+As an alternative to Windows, you can use a cloud IDE such as [Cloud 9](https://c9.io) for a relatively painless setup. If you take this route, please ensure to upgrade to the latest Python version with the following. (This is to avoid InsecurePlatformWarning that arise when the default Python 2.7.6 is used).  
```

Comment:
minor nits:

Can we put `InsecurePlatformWarning` in backticks? That way it will be styled like code, e.g., \`InsecurePlatformWarning\` 

Maybe instead of "This is to avoid InsecurePlatformWarning that arise" possibly "This is to avoid InsecurePlatformWarning errors that arise"?